### PR TITLE
Aggregate initialize LEDs to simplify code.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: LEDControl.cpp [hunk 1]
```diff
@@ -5,39 +5,44 @@
 /// class for individual LED with non-blocking control
 //
 
-LEDControl::LEDControl() {
+LEDControl::LEDControl() 
+    : _state(LOW)
+    , _flash(false)
```
Comment (MartinDaCosta53) [inline LEDControl.cpp:10]: This is a comment

File: LEDControl.cpp [hunk 2]
```diff
@@ -5,39 +5,44 @@
...
-  _state = LOW;
-  _flash = false;
-  _lastTime = 0UL;
+LEDControl::LEDControl(byte pin)
+    : _state(LOW)
+    , _flash(false)
+    , _lastTime(0UL)
+    , _pin(pin)
+{
+	pinMode(_pin, OUTPUT);
```
Comment (ghost) [inline LEDControl.cpp:21]: This is a constructor for sven.

File: LEDControl.cpp (hunk 1 above)
Comment (ghost) [reply LEDControl.cpp:10]: Thank you.

File: LEDControl.cpp (hunk 2 above)
Comment (MartinDaCosta53) [reply LEDControl.cpp:21]: And everyone else

Comment (ghost) [reply LEDControl.cpp:21]: fgjgdfjhm

**Pull request closed without merging**